docs/modules/ROOT/pages/references/architecture/upgrade_controller.adoc

```diff
@@ -300,21 +304,26 @@ It contains a snapshot of the most recent version in the `.status.availableUpdat
 apiVersion: managedupgrade.appuio.io/v1beta1
 kind: UpgradeJob
 metadata:
-  name: cluster-upgrade-2021-01-01T22:00:00+01:00
+  name: cluster-upgrade-1609531200-ef11c47 <1>
 spec:
-  upgradeAt: "2021-01-01T22:00:00+01:00" <1>
-  desiredVersion: <2>
+  startAfter: "2021-01-01T22:00:00+01:00" <2>
+  startBefore: "2021-01-01T23:00:00+01:00" <3>
```

Comment:
The example config has {{maxUpgradeStartDelay: 2h}}, but here we have an upgrade start window of 1h. Is this intentional, and if so, what's the logic to derive a start window of 1h from a config value of 2h?

Reply:
this was a typo, fixed